commands/slash/247.js

```diff
@@ -41,6 +41,7 @@ const command = new SlashCommand()
       const embed = client.Embed(`✅ | **24/7 mode is now on.**`);
       return interaction.reply({ embeds: [embed] });
     }
+      if(err) console.log(err) && client.errorEmbed("An error has occurred, please vist console for more information!")
```

Comment:
*Remove this line entirely there is no use for this

Reply:
Below it was a text saying it has no checks for errors so I added one

Reply:
it does has checks for errros?

Reply:
oh I see what you're talking about, that one has been solved.

Reply:
still remove that tho, you didn't define "err" therefore it's just a zombie code

Reply:
alr np